Description/hi-IN.md

```diff
@@ -0,0 +1,89 @@
+# हिंदी (इंडिया) 
+
+एक Fabric पर बना हुआ मोडपैक है माइनकक्राफ्ट के लिए जो परफॉर्मांस और ग्राफिक्स को बेहतर करने पर अपना ध्यान केंद्रित करता है।
+
+[![अभी की समीक्षा](https://img.youtube.com/vi/K90gsbmhf3w/maxresdefault.jpg)](https://www.youtube.com/watch?v=K90gsbmhf3w)
+
+### परफॉर्मांस
+
+Fabulously Optimized ज्यादतर OptiFine से बेहतर साबित होता है क्योंकि उसमे [आधुनिक पेरफॉर्मांस मॉड्स का इस्तेमाल किया गया है][1] जैसे Sodium और Lithium.
+
+### OptiFine के साथ समानता
+
+Fabulously Optimized [OptiFine के लगभग सभी विभिन्न विशेषताओं को स्पोर्ट करता है][2], जिसमे shaders, connected textures और zoom आते हैं. [एक मुफ्त cape पाएं!][3]
+
+### कम बदलाव
+
+Fabulously Optimized [मे कुछ बदलाव हैं][4] जो गमे के साधारण भावों को बिल्कुल नहीं बदलता, जैसे ज्यादा चैट हिस्ट्री, विंडोवद् फुल स्क्रीन मोड और कई तरह की गलत चीजों को सही करने जैसे आम मुद्दों के लिए.
+
+### आधुनिक
+
+Fabulously Optimized [जल्दी उपदेट्स पाता है][5] और नये गमे वेरसनस मे उपडेट होने वाले modpacks मे से वन ऑफ दी फर्स्ट modpack है.
+
+### कहीं भी चल सकता है
+
+Fabulously Optimized को ऐसे बनाया गया है की व किसी भी जगह चल सकता है, [चार launchers पे चलता है, तीन sites पे उपलब्ध है][6] और [कई सारी भाषाओं मे उपलब्ध है][7].
+
+### खुला विकास
+
+Fabulously Optimized [open-source है][8] और उसके कई सारे mods भी. इसका विकास जनता के विचारों पर होता है, mod developers के साथ काम करके सबके लिए एक अच्छा वातावरन बनाने की कोशिस की जाती है.
+
+### Fabric पर बना है
+
+Forge के मुकाबले, Fabric जल्दी उपडटे और लोड होता है, बड़ी बड़ी फ़ाइलें नहीं शामिल हैं और फिर भी अच्छा mod support हैं. [(हाँ, Quilt पर सोच-विचार किया जारहा है.)][9]
+
+### मददगार जनता
+
+हमारे पास एक दोस्तानी सी जनता है जो modpack के बारे मे बात करते हैं और एक दूसरे की मदत करते हैं. [हमसे जुड़ें!][10]
+
+## प्रशन और उत्तर
+
+> **प्रशन**: कैसे इंस्टाल करें? उपडेट करें? mods बदलें? और कुछ?
+> 
+> **उत्तर**: [इन सबके जवाब हमारी wiki पर उपलब्ध हैं.][11]
+
+
+> **प्रशन**: कुछ टूट गया! तुम्हे कुछ जोड़ना है!
+>
+> **उत्तर**: [हमारे GitHub पेज पर डालें.][8]
+
+
+> **प्रशन**: कौनसे mods उपलब्ध हैं? 
+>
+> **उत्तर**: किसी भी संस्करण को दबाकर नीचे जाएं [या नीचे दी गयी टेबल देखें][12].
+
+
+> **प्रशन**: मुझे ये अच्छा लगा. में इसे बनाने मे कैसे मदत कर सकता हूँ?
+> 
+> **उत्तर**: [मदत करने के तरीके यहाँ पर दिये हुए हैं.][16] शुक्रिया!
+
+
+> **प्रशन**: क्या मुझे ये मोडपैक अपने सर्वर पे भी चाहिए होगा?
+> 
+> **उत्तर**: नहीं, ये किसी भी सर्वर पर काम करता है जो vannila clients को support करते हैं. [ज्यादा विस्तृत मे उत्तर और सेर्वर से जुड़े टिप्स यहाँ मौजूद हैं.][13] 
```

Comment:
> vannila clients को support 

fix vanilla* typo at least, but I believe at least "support" can be translated too